Dell-Inspiron_15_3535-20231209-174814.yml

```diff
@@ -0,0 +1,58 @@
+---
+layout:
+  'hcl'
+type:
+  'laptop'
+hvm:
+  'yes'
+iommu:
+  'yes'
+slat:
+  'yes'
+tpm:
+  '2.0'
+remap:
+  'yes'
+brand: |
+  Dell
+model: |
+  Inspiron 15 3535
+bios: |
+  1.2.0
+cpu: |
+  AMD Ryzen 5 7530U with Radeon Graphics
+cpu-short: |
+  Ryzen 5 7530U
+chipset: |
+  Advanced Micro Devices, Inc. [AMD] Renoir/Cezanne Root Complex [1022:1630]
+chipset-short: |
+  AMD
+gpu: |
+  Advanced Micro Devices, Inc. [AMD/ATI] Barcelo [1002:15e7] (rev c5) (prog-if 00 [VGA controller])
+gpu-short: |
+  AMD
+network: |
+  Realtek Semiconductor Co., Ltd. RTL8821CE 802.11ac PCIe Wireless Network Adapter [10ec:c821]
+memory: |
+  15720
+scsi: |
+
+usb: |
+  2
+certified:
+  'no'
+versions:
+  - works:
+      partial
+    qubes: |
+      R4.2.0-rc5
+    xen: |
+      4.17.2
+    kernel: |
+      6.6.2-1
+    remark: |
+      Wifi does not work
+    credit: |
+      @qos_newbie
+    link: |
+      https://forum.qubes-os.org/t/laptop-dell-inspiron-15-3535/22679
```

Comment:
I've always shortened these URLs, but that's not necessary.
`https://forum.qubes-os.org/t/22679`